### PR TITLE
Drop unused labels

### PR DESCRIPTION
**Pull request by vikaschoudhary16**
Depends on: https://github.com/openshift/cluster-api-provider-aws/pull/182

### CODE REVIEW

File: pkg/controller/machinehealthcheck/machinehealthcheck_controller.go [hunk 1]
```diff
@@ -331,19 +331,13 @@ func hasMatchingLabels(machineHealthCheck *healthcheckingv1alpha1.MachineHealthC
...
 
 func isMaster(machine mapiv1.Machine, client client.Client) bool {
-	machineMasterLabels := []string{
-		"machine.openshift.io/cluster-api-machine-role",
-		"machine.openshift.io/cluster-api-machine-type",
-		"sigs.k8s.io/cluster-api-machine-role",
-		"sigs.k8s.io/cluster-api-machine-type",
-	}
-	nodeMasterLabels := []string{
+	masterLabels := []string{
```
Comment (cynepco3hahue) [inline pkg/controller/machinehealthcheck/machinehealthcheck_controller.go]: Does it make sense to check machine labels at all?
IMHO the only proper way is to check node labels that machine reference to.

Comment (enxebre) [reply pkg/controller/machinehealthcheck/machinehealthcheck_controller.go]: I agree. @vikaschoudhary16 as part of this effort can we drop labels here https://github.com/openshift/installer/tree/master/pkg/asset/machines/aws
And update https://github.com/openshift/origin/pull/22090/files#r267287918 to fetch from node accordingly cc @ingvagabund @derekwaynecarr

**Pull request merged**